ui/lib/pki/addon/templates/issuers/issuer/sign.hbs

```diff
@@ -1 +1,13 @@
-route: issuers.issuer.sign
\ No newline at end of file
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{this.breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-pki-issuer-page-title>
+      <Icon @name="file-text" @size="24" class="has-text-grey-light" />
+      Sign intermediate
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<PkiSignIntermediateForm @model={{this.model}} @onCancel={{transition-to "vault.cluster.secrets.backend.pki.issuers"}} />
```

Comment:
It seems like we'd want to go back to the issuer we were viewing before clicking "Sign intermediate" and not the index page - but maybe this was already discussed and decided by another conversation!